docker-compose.yaml

```diff
@@ -1,48 +1,34 @@
 version: '3'
+
 services:
   ffc-demo-claim-service:
     build: .
     image: ffc-demo-claim-service
-    container_name: ffc-demo-claim-service
     depends_on:
       - ffc-demo-claim-postgres
+    env_file:
+      - variables/message-queues.env
+      - variables/postgres.env
     environment:
-      POSTGRES_DB: ffc_demo_claims
-      POSTGRES_PASSWORD: ppp
-      POSTGRES_USERNAME: postgres
       POSTGRES_HOST: ffc-demo-claim-postgres
       POSTGRES_PORT: 5432
       MESSAGE_QUEUE_HOST : host.docker.internal
       MESSAGE_QUEUE_PORT : 5672
-      CALCULATION_QUEUE_ADDRESS : calculation
-      CALCULATION_QUEUE_USER : artemis
-      CALCULATION_QUEUE_PASSWORD : artemis
-      SCHEDULE_QUEUE_ADDRESS : schedule
-      SCHEDULE_QUEUE_USER : artemis
-      SCHEDULE_QUEUE_PASSWORD : artemis
     networks:
-      - claim-postgres
-    ports:
-      - "3003:3003"
+      - ffc-demo-claim
     volumes:
       - node_modules:/usr/src/app/node_modules
       - .:/usr/src/app
 
   ffc-demo-claim-postgres:
-    container_name: ffc-demo-claim-postgres
     image: postgres:11.4-alpine
-    environment:
-      POSTGRES_DB: ffc_demo_claims
-      POSTGRES_PASSWORD: ppp
-      POSTGRES_USERNAME: postgres
+    env_file:
+      - variables/postgres.env

```

Comment:
I like the use of `env_file` to reduce duplication and guarantee that the two containers maintain identical settings for postgres.